### DISCUSSION TITLE
is it possible to have output inverted, for HUD projection on windshield?

### DISCUSSION THREAD

**Discussion by greymda** [alexanderlavrushko/BLE-HUD-navigation-ESP32#6] [Ideas]
basically, is it possible to have output inverted, for HUD projection on windshield?
something like this, but without using smartphone.

![sygic-hud-1](https://user-images.githubusercontent.com/9991596/125203747-8599b100-e282-11eb-9d9c-ae39fbc9610b.jpeg)

**Comment by alexanderlavrushko** [comment]
Currently mirroring is not implemented, but I think it would be nice to have, maybe I'll find time for it this month.

**Comment by alexanderlavrushko** [comment]
It's done, try setting ```DISPLAY_MIRRORED``` and ```DISPLAY_ROTATION```

**Reply by greymda** [reply]
that was FAST! will definitely try when i receive my ttgo t-display board! thank you!
